<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_pic" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>481</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始采集</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_CamList">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接相机</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="expose">
    <property name="geometry">
     <rect>
      <x>752</x>
      <y>224</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>224</y>
      <width>57</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_expose">
    <property name="geometry">
     <rect>
      <x>633</x>
      <y>227</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderExpose">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>297</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
